FiveCalls/FiveCalls/LocationSheet.swift

```diff
@@ -0,0 +1,141 @@
+//
+//  LocationSheet.swift
+//  FiveCalls
+//
+//  Created by Nick O'Neill on 8/7/23.
+//  Copyright © 2023 5calls. All rights reserved.
+//
+
+import SwiftUI
+import CoreLocation
+
+struct LocationSheet: View {
+    @Environment(\.dismiss) var dismiss
+    
+    let location: NewUserLocation?
```

Comment:
It doesn't look like this is used. Did you plan to populate it when already chosen?